[docs] [#11472] Removed manual transaction table creation from geo-partioning example (#11474)
yugabyte/yugabyte-db: docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md

![Row-level geo-partitioning](/images/explore/multi-region-deployments/geo-partitioning-2.png)

{{< tip title="Region-local transaction table" >}}

> When we create tables using a tablespace with a placement set, YugabyteDB automatically creates a transaction table under the tablespace if one does not yet exist, with a name like `system.transactions_90141438-f42c-4a39-8a12-4072c1216d46`.

Reviewer: (I want to move all our docs to second-person, but at least let's get the notes that way :) )
```suggestion
When you create tables using a tablespace with a placement set, YugabyteDB automatically creates a transaction table under the tablespace if one doesn't yet exist, with a name like `system.transactions_90141438-f42c-4a39-8a12-4072c1216d46`.
```

ERROR:  Illegal state: Nonlocal tablet accessed in local transaction: tablet c5a611afd571455e80450bd553a24a64: . Errors from tablet servers: [Illegal state (yb/client/transaction.cc:284): Nonlocal tablet accessed in local transaction: tablet c5a611afd571455e80450bd553a24a64]
```

> Because we have a tablespace with placement set to the us-west-2 region, YugabyteDB assumes by default that we want to run a transaction local to that region (using the transaction status table automatically created for the `us_west_2_tablespace` created in Step 1), but such a transaction cannot modify data outside of us-west-2.

Reviewer: Too many "created" in short order :)

```suggestion
Because we have a tablespace with placement set to the us-west-2 region, YugabyteDB assumes by default that we want to run a transaction local to that region (using the automatically-generated transaction status table for the `us_west_2_tablespace` created in Step 1), but such a transaction cannot modify data outside of us-west-2.
```
